sunpy/map/tests/test_compositemap.py

```diff
@@ -64,7 +64,22 @@ def test_plot_composite_map_colors(composite_test_map):
     composite_test_map.plot(colors='red')
 
 
-def test_plot_composite_map_mplkwargs(composite_test_map):
+def test_plot_composite_map_updated_args(composite_test_map, simple_map):
+    levels = [-1000, -500, -250, 250, 500, 1000] * u.G
+    composite_test_map.set_levels(index=1, levels=levels)
```

Comment:
What is this testing?

Reply:
> I included `contour_args = {'norm': 'log'}` to confirm whether the original `contour_args` from `draw_contours` take precedence over `plot_settings`. 

Testing if the right contours args will be finally passed down to mlp contour